documentation/README_DE.md

```diff
@@ -1,48 +1,54 @@
 <p align="center">
-    <img src="https://github.com/acemod/ACE3/blob/master/extras/assets/logo/black/ACE3-Logo.jpg" height="80" />
+    <img src="https://github.com/acemod/ACE3/blob/master/extras/assets/logo/black/ACE3-Logo.jpg" width="480">
 </p>
 <p align="center">
     <a href="https://github.com/acemod/ACE3/releases">
-        <img src="http://img.shields.io/badge/release-3.0-green.svg?style=flat" alt="ACE version">
+        <img src="https://img.shields.io/badge/Version-3.2.1-blue.svg" alt="ACE3 Version">
     </a>
-    <a href="#">
-        <img src="http://img.shields.io/badge/download-22_MB-blue.svg?style=flat" alt="ACE download">
+    <a href="https://github.com/acemod/ACE3/releases/download/v3.2.1/ace3_3.2.1.zip">
+        <img src="http://img.shields.io/badge/Download-56.5_MB-green.svg" alt="ACE3 Download">
     </a>
     <a href="https://github.com/acemod/ACE3/issues">
-        <img src="http://img.shields.io/github/issues/acemod/ACE3.svg?style=flat" alt="ACE issues">
+        <img src="http://img.shields.io/github/issues-raw/acemod/ACE3.svg?label=Issues" alt="ACE3 Issues">
+    </a>
+    <a href="https://forums.bistudio.com/topic/181341-ace3-a-collaborative-merger-between-agm-cse-and-ace/?p=2859670">
+        <img src="https://img.shields.io/badge/BIF-Thread-lightgrey.svg" alt="BIF Thread">
     </a>
     <a href="https://github.com/acemod/ACE3/blob/master/LICENSE">
-        <img src="http://img.shields.io/badge/license-GPLv2-red.svg?style=flat" alt="ACE license">
+        <img src="http://img.shields.io/badge/License-GPLv2-red.svg" alt="ACE3 License">
     </a>
 </p>
 <p align="center"><sup><strong>Benötigt eine Aktuelle Version von <a href="http://www.armaholic.com/page.php?id=18767">CBA A3</a> | <a href="#">BIF thread</a></strong></sup></p>
+<p align="center"><sup><strong>Requires the latest version of <a href="http://www.armaholic.com/page.php?id=18767">CBA A3</a>. Besucht uns auf <a href="https://www.facebook.com/ACE3Mod">Facebook</a> | <a href="https://www.youtube.com/c/ACE3Mod">YouTube</a> | <a href="https://twitter.com/ACE3Mod">Twitter</a> | <a href="http://www.reddit.com/r/arma/search?q=ACE&restrict_sr=on&sort=new&t=all">Reddit</a></strong></sup></p>
```

Comment:
Line 21 and 22 look wierd. There is a link to a BIF thread that should not be here and the text on line 22 is in English. Line 21 has the German version.